docker-compose.yml

```diff
@@ -78,11 +79,12 @@ services:
       - CD_EVENTS=events
       - CD_BADGES=badges
       - POSTGRES_HOST=db
+      - CDF_ADMINS=admin@coderdojo.org,daniel@coderdojo.org
```

Comment:
I get admin but should Daniel be Commited

Reply:
I'm not sure we can cast Daniel as Code

Reply:
Where does zen end and Daniel begin?

Reply:
Are you trying to write a drama or a love story ? :p